<script>
    console.log('Task 1');
    try{
        console.log(z);
    }
    catch{
        console.log('Some Error occured and Handled..!');
    }
    console.log('Task 2');
</script>